packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -237,6 +240,12 @@ export default declare((api, options) => {
               replacementPath.get("alternate"),
             );
           }
+
+          needsRequeue = true;
+        }
+
+        if (needsRequeue) {
+          parentPath.requeue();
```

Comment:
I _think_ we should requeue `replacementPath` and not `parentPath`, since they are not always the same thing.

Reply:
That was my first thought too, but there's some non obvious thing going on, still digging further.

Reply:
Oh ok. Wdyt about merging this PR since "it works", and then revisit for another release?

Reply:
Oh ok. Do you think that it's better to merge this PR since "it works", and then revisit for another release? Or should we wait?